libplatsupport/src/plat/rk3568/serial.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
```

Comment:
Please document what serial device is used and its compat string (and double check there isn't a compatible driver anywhere yet, if possible).